scripts/utils/logs.go

```diff
@@ -0,0 +1,153 @@
+// Author: Haoyuan Ma <flyinghorse0510@zju.edu.cn>
+package utils
+
+import (
+	"fmt"
+	"log"
```

Comment:
use `logrus` instead of log like in the rest of the repo. It already has LogError, LogWarn, etc. 
Deprecate redundant helper functions